third_party/accessibility/ax/platform/ax_platform_node_textrangeprovider_win_unittest.cc

```diff
@@ -5109,19 +5108,17 @@ TEST_F(AXPlatformNodeTextRangeProviderTest, TestITextRangeProviderFindText) {
   // Test Leaf kStaticText search.
   GetTextRangeProviderFromTextNode(range, root_node->children()[0]);
   EXPECT_UIA_FIND_TEXT(range, L"some text", false, owner);
-  // Some expectations like the one below are currently skipped until we can
-  // implement ignoreCase in FindText.
-  // EXPECT_UIA_FIND_TEXT(range, L"SoMe TeXt", false, owner);
+  EXPECT_UIA_FIND_TEXT(range, L"SoMe TeXt", true, owner);
   GetTextRangeProviderFromTextNode(range, root_node->children()[1]);
   EXPECT_UIA_FIND_TEXT(range, L"more", false, owner);
-  // EXPECT_UIA_FIND_TEXT(range, L"MoRe", true, owner);
+  EXPECT_UIA_FIND_TEXT(range, L"MoRe", true, owner);
 
   // Test searching for leaf content from ancestor.
   GetTextRangeProviderFromTextNode(range, root_node);
   EXPECT_UIA_FIND_TEXT(range, L"some text", false, owner);
-  // EXPECT_UIA_FIND_TEXT(range, L"SoMe TeXt", true, owner);
+  EXPECT_UIA_FIND_TEXT(range, L"SoMe TeXt", true, owner);
   EXPECT_UIA_FIND_TEXT(range, L"more text", false, owner);
-  // EXPECT_UIA_FIND_TEXT(range, L"MoRe TeXt", true, owner);
+  EXPECT_UIA_FIND_TEXT(range, L"MoRe TeXt", true, owner);
   EXPECT_UIA_FIND_TEXT(range, L"more", false, owner);
```

Comment:
If we adopt Chromium's ICU-based StringSearch implementation, we should add further tests like checking that `eleve` is found in the string `élève`.